daliuge-engine/test/integrate/example_split.py

```diff
@@ -5,6 +5,8 @@
 To run it standalone, change the directories, which are now hardcoded
 """
 
+# pylint: skip-file
+
```

Comment:
I am skipping this file completely because it is using `mstransform`, which is a CASA method that, at some point in time, was available in this code (although I'm not sure how/where it ever ran). This file is broken/un-runable code, but I don't want to remove it from the code base as I do not know it's use case. A decision can be made either now or in a future PR about what to do with it.